types/three/three-fbxloader.d.ts

```diff
@@ -1,14 +1,17 @@
 // https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/FBXLoader.js
 
 import { LoadingManager } from "./three-core";
-import {Group} from "./three-core";
+import { AnimationClip, Group } from "./three-core";
 
+interface ILoadedModel extends Group {
```

Comment:
The implementation seems to refer to its return value as a `sceneGraph` so maybe something like `FbxSceneGraph` would be a more appropriate name? Up to you though.

Reply:
FbxSceneGraph is much better